src/lib/components/file-browser/terra-frontend-storage.service.ts

```diff
@@ -164,27 +140,20 @@ export class TerraFrontendStorageService extends TerraBaseMetadataStorageService
         ));
     }
 
-    public deleteFiles(keyList:Array<string>):Observable<void>
+    public deleteFiles(keyList:Array<string>):Observable<void | Object>
     {
-        this.setAuthorization();
-        return this.mapRequest(
-            this.http.delete(
-                '/rest/storage/frontend/files?' + keyList.map((key:string):string => 'keyList[]=' + key).join('&'),
-                {
-                    headers: this.headers
-                }
-            )
-        ).pipe(tap(():void =>
-            {
-                keyList.forEach((key:string):void => this._storageList.root.removeChild(key));
-                this._storageListSubject.next(this._storageList);
-            },
-            ():void =>
-            {
-                this._storageInitialized = false;
-                this._storageListSubject.next(null);
-            }
-        ));
+        return this.http.delete('/rest/storage/frontend/files?' + keyList.map((key:string):string => 'keyList[]=' + key).join('&'))
```

Comment:
Dann brauchst du auch oben nicht das `| Object` bei return type
```suggestion
        return this.http.delete<void>('/rest/storage/frontend/files?' + keyList.map((key:string):string => 'keyList[]=' + key).join('&'))
```